user-manual.adoc

```diff
@@ -46,6 +46,8 @@ play {
 }
 ```
 
+Be aware that the get() method of your CustomSMTPConfigurationProvider will be called multiple times.
```

Comment:
To draw attention, you should use admonition: http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#paragraphs

```asciidoc
IMPORTANT: The `get()` method of your CustomSMTPConfigurationProvider will be called each time you send an email.
```